Source/m_menu.c

```diff
@@ -3886,11 +3894,17 @@ setup_menu_t gen_settings3[] = { // General Settings screen3
   {"Game speed, percentage of normal", S_NUM|S_PRGWARN, m_null, M_X,
    M_Y + general_realtic*M_SPC, {"realtic_clock_rate"}},
 
+  {"Show demo progress bar", S_YESNO, m_null, M_X,
+   M_Y + general_demobar*M_SPC, {"demobar"}},
+
   {"Default skill level", S_CHOICE|S_LEVWARN, m_null, M_X,
    M_Y + general_skill*M_SPC, {"default_skill"}, 0, NULL, default_skill_strings},
 
-  {"Show demo progress bar", S_YESNO, m_null, M_X,
-   M_Y + general_demobar*M_SPC, {"demobar"}},
+  {"Show ENDOOM screen", S_CHOICE, m_null, M_X,
+   M_Y + general_endoom*M_SPC, {"show_endoom"}, 0, NULL, default_endoom_strings},
+
+  {"Network Player Name", S_CHAT, m_null, M_X,
```

Comment:
I like this rearrange, thanks!
Two small issues with `net_player_name`: the default reset button in the chat menu also resets it here, also hu_font only supports upper case. No big deal, but it might be easier to just remove it.